sdk/communication/Azure.Communication.CallingServer/src/CallingServerClient.cs

```diff
@@ -536,13 +536,25 @@ public virtual Response ResumeRecording(string recordingId, CancellationToken ca
         public virtual async Task<Response<Stream>> DownloadStreamingAsync(
             Uri sourceEndpoint,
             HttpRange range = default,
-            CancellationToken cancellationToken = default) =>
-            await _contentDownloader.DownloadStreamingInternal(
+            CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(CallingServerClient)}.{nameof(DownloadStreaming)}");
```

Comment:
Are we ignoring the `Async` suffix of the methods? How will we know which method is the failing one if two methods share the same scope naming?

Reply:
I have the same question on how DiagnosticScope works, @zihzhan-msft Can you please throw some info on this?

Reply:
https://teams.microsoft.com/l/message/19:97e6cbbb188d4f8e9e7ce0f189f362ae@thread.skype/1638815675144?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1638815675144&teamName=Azure%20SDK&channelName=Azure%20Core&createdTime=1638815675144

Create a thread in Azure Core channel for the scope name between Sync and Async methid.

Reply:
https://teams.microsoft.com/l/message/19:97e6cbbb188d4f8e9e7ce0f189f362ae@thread.skype/1638815675144?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1638815675144&teamName=Azure%20SDK&channelName=Azure%20Core&createdTime=1638815675144

Create a thread in Azure Core channel for the scope name between Sync and Async methid.

Reply:
@ravithanneeru Per the answer in the thread, we should remove `Async` postfix for all the api operations, can you help on that?

Reply:
@zihzhan-msft we never used Async in any of the CreateScope calls.